cost/azure/rightsize_sql_instances/azure_rightsize_sql_instances.pt

```diff
@@ -508,14 +542,23 @@ datasource "ds_azure_sql_database_metrics" do
     encoding "json"
     field "value", jmes_path(response, "value")
     field "id", val(iter_item, "id")
-    field "region", val(iter_item, "region")
     field "name", val(iter_item, "name")
-    field "kind", val(iter_item, "kind")
     field "type", val(iter_item, "type")
+    field "kind", val(iter_item, "kind")
     field "sku", val(iter_item, "sku")
     field "tags", val(iter_item, "tags")
+    field "createdTime", val(iter_item, "createdTime")
+    field "status", val(iter_item, "status")
+    field "region", val(iter_item, "region")
+    field "serverId", val(iter_item, "serverId")
+    field "serverKind", val(iter_item, "serverKind")
+    field "serverName", val(iter_item, "serverName")
+    field "serverTags", val(iter_item, "serverTags")
+    field "serverFQDN", val(iter_item, "serverFQDN")
+    field "serverState", val(iter_item, "serverState")
+    field "serverVersion", val(iter_item, "serverVersion")
     field "resourceGroup", val(iter_item, "resourceGroup")
-    field "subscriptionId", val(iter_item, "subscriptionId")
```

Comment:
for consistency in other policies we have field "id", jmes_path(col_item, "subscriptionId"). Do we want to change this to ID?

Reply:
It's not consistent from policy to policy already and it ultimately doesn't actually matter IMO